configuration/ibm/50001000.json

```diff
@@ -1122,12 +1122,29 @@
                         }
                     }
                 },
+                "postAction": {
+                    "deletion": {
```

Comment:
In case of lcd panel why this can't be a part of preAction? Here we will not have key clash I guess?

Reply:
There will be no key clash, can be part of preAction. I thought for all the FRUs we can have the same flow. Like 
1. if led is present unbind it 
2. clear the VPD data under PIM
3. Then unbind the device driver and disable the GPIO line